docs/spec_draft.md

```diff
@@ -399,6 +400,51 @@ dimensions `k` in `operand`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.case
+
+### Semantics
+
+Produces the output from executing exactly one `function` from `branches`
```

Comment:
Given that the word "function" here is not used to refer to a input (the input is called "branches") or to refer to a type, let's remove the backticks. Also, the IfOp spec says "branch" here.